com.reprezen.swagedit.openapi3.tests/src/com/reprezen/swagedit/openapi3/editor/hyperlinks/HyperlinkDetectorTest.xtend

```diff
@@ -0,0 +1,83 @@
+package com.reprezen.swagedit.openapi3.editor.hyperlinks
```

Comment:
A copyright header?

Reply:
The same for `HyperlinkDetectorTest`, `LinkOperationHyperlinkDetector `, and  `SecuritySchemeHyperlinkDetector `